app/controllers/provider/admin/account/invitations_controller.rb

```diff
@@ -23,11 +21,7 @@ def resend
     @invitation.resend
 
     respond_to do |format|
-      format.html do
-        flash[:success] = 'Invitation will be resent soon.'
-        redirect_to provider_admin_account_invitations_path
-      end
-      format.xml  { head :ok }
```

Comment:
Are you sure this is not used anymore?

Reply:
UPDATE: After some digging I can't really justify removing this response so I will restore it and add a comment for posterity.